google/cloud/bigquery/model.py

```diff
@@ -305,9 +305,7 @@ def from_api_repr(cls, resource):
             start_time = datetime_helpers.from_microseconds(1e3 * float(start_time))
             training_run["startTime"] = datetime_helpers.to_rfc3339(start_time)
 
-        this._proto = json_format.ParseDict(
-            resource, types.Model()._pb, ignore_unknown_fields=True
-        )
+        this._proto = types.Model.from_json(json.dumps(resource))
```

Comment:
Can we pass `resource` to the `Model` constructor here?

https://github.com/googleapis/proto-plus-python/issues/152#issuecomment-712290339

Reply:
We'll still want https://github.com/googleapis/proto-plus-python/issues/153 to support `ignore_unknown_fields`, though.

Reply:
I already had tried that during my testing. It gives a KeyError on the metaclass during `__init__`, right here:

https://github.com/googleapis/proto-plus-python/blob/master/proto/message.py#L425

I was using this code:

    this._proto = types.Model(resource)

Reply:
We should be able to update this to `types.Model(resource, ignore_unknown_fields=True)` now.